account-kit/smart-contracts/src/light-account/accounts/predictAddress.ts

```diff
@@ -90,6 +86,57 @@ export function predictLightAccountAddress({
   }
 }
 
+export type PredictMultiOwnerLightAccountAddressParams = {
+  factoryAddress: Address;
+  salt: bigint;
+  ownerAddresses: Address[];
+  // There's just one version of the MultiOwnerLightAccount for now, so skip requiring the version as a parameter.
+};
+
+// Note: assumes the owner addresses are already deduped, sorted in ascending order, and have the signer address included.
+export function predictMultiOwnerLightAccountAddress({
+  factoryAddress,
+  salt,
+  ownerAddresses,
+}: PredictMultiOwnerLightAccountAddressParams): Address {
+  const implementationAddress =
+    // If we aren't using the default factory address, we compute the implementation address from the factory's `create` deployment.
+    // This is accurate for both LA v1 and v2 factories. If we are using the default factory address, we use the implementation address from the registry.
+    factoryAddress !==
+    AccountVersionRegistry.MultiOwnerLightAccount["v2.0.0"].addresses.default
+      .factory
+      ? getContractAddress({
```

Comment:
is this not async?

Reply:
nope! https://viem.sh/docs/utilities/getContractAddress